### DISCUSSION TITLE
Displaying data from last trial

### DISCUSSION THREAD

**Discussion by revatis2** [jspsych/jsPsych#987] [Q&A] [answered]
Hello!
I am trying to display RT from the last trial in the next trial as stimulus. I am getting this error: jsPsych.data.getLastTrialData() is undefined. 
Where am I going wrong?
My code:
`var stage_1 = {
      type: 'html-keyboard-response',
      stimulus: 'LEFT OR RIGHT',
      choices: [37, 38]
    }
    timeline.push(stage_1)`

    var disp = {
      type: 'html-keyboard-response',
      stimulus: jsPsych.data.getLastTrialData().values()[0].rt,
      choices:[32]
    }
    timeline.push(disp)

    jsPsych.init({
      timeline: timeline,
      on_finish: function(){
        jsPsych.data.displayData()
      }
    })
`

**Comment by vijaymarupudi** [comment, marked as answer]
You'll want to put the stimulus for the second trial in a [function](https://www.jspsych.org/overview/trial/#dynamic-parameters) as it depends on runtime information.

**Comment by revatis2** [comment]
Thanks, works perfectly!
